LunrCoreLmdb/LmdbIndex.cs

```diff
@@ -6,6 +6,8 @@
 using System.Threading.Tasks;
 using LightningDB;
 using Lunr;
+using Lunr.Globalization.fr;
+using Lunr.Globalization.it;
```

Comment:
Also seems to be differing only by white space and `using`s.

Reply:
Detritus.